src/types/publication.types.ts

```diff
@@ -73,8 +80,21 @@ interface PublicationEntityDescription {
   language: LanguageCodes;
   projects: Project[];
   publicationType: ReferenceType | '';
+  publicationSubtype: JournalArticleType | ReportType | DegreeType | BookType | '';
   contributors: Contributor[];
   doiUrl: string;
+  publisher: Publisher;
+  volume: string;
+  issue: string;
+  pagesFrom: string;
+  pagesTo: string;
+  peerReview: boolean;
+  articleNumber: string;
+  isbn: string;
+  numberOfPages: string;
+  series: Publisher;
+  specialization: string;
```

Comment:
hva er specialization?

Reply:
studieretning. ser ikke ut som vi har noen felter for å sette verdi her enda da, så kunne kanskje bare fjerna den også?

Reply:
ah ok. tja, kan sikkert la den stå hvis den ikke gjør noe skade?

Reply:
nei, i praksis en placeholder atm

Reply:
da kan den sikkert bare stå